src/Views/elevator.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from "react";
+import Diagram from "../Components/Diagram/Diagram";
+import PadBtn from "../Components/PadBtn/PadBtn";
+import { Grid } from "../styles";
+
+interface IProps {}
+
+interface IState {
+  count: number;
+  floors: string[];
+  moving: string;
+}
+
+class Elevator extends React.Component<IProps, IState> {
+  constructor(props: IState) {
+    super(props);
+
+    this.state = {
+      count: 0,
+      floors: ["G", "1", "2", "3", "4"],
+      moving: "right",
+      //isGoingUp: true,
+      //idle: true,
+      //queue: [],
+    };
+  }
+
+  //changeDirection
+  //reverseQueue
+
+  callAndGo = (btnFloor: number, goUp: boolean) => {
+    this.setState(() => ({
+      moving: "left",
+    }));
+    setTimeout(() => {
```

Comment:
All the timeOuts will be removed in the next PR. So they don't need to me reviewed. They were used so I could test how the elevator would interact with the buttons and for better visual presentation on the UI